### PR TITLE
[Library updated] Authors Removed

### PR DESCRIPTION
**Pull request by Tiansing**
All authors mentioned were successfully removed!

### CODE REVIEW

File: tests/Abuse/AbuseTest.php
Comment (Meldiron) [inline tests/Abuse/AbuseTest.php:9]: Let's remove the whole comment blocks, please.

**Review by Meldiron — CHANGES_REQUESTED** [review]
One tiny comment 🤏 I can mark as approved once resolved.

Also, looks like you done changes to composer lockfile. Can we revert those? This file is auto-generated so authors there done need to be touched - it will get re-added anyway.

**Comment by Tiansing** [comment]
> 

@Meldiron Done resolving this issue thanks

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Review by stnguyen90 — DISMISSED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: composer.lock
Comment (Tiansing) [reply composer.lock:51]: @stnguyen90 noted that!

Comment (stnguyen90) [reply composer.lock:51]: @Tiansing please make sure to revert those changes of yours and resolve any merge conflicts if there are any.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Comment by Tiansing** [comment]
> https://github.com/orgs/appwrite/invitation

Hi @christyjacob4, I can't access the URL you've provided for me to accept the github org invitation. Please send me an updated invitation link for GitHub appwrite org, I would love to join in your community

**Pull request merged**
